vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -369,4 +369,8 @@ def initialize_dummy_weights(
     """
     for param in model.state_dict().values():
         if torch.is_floating_point(param):
-            param.data.uniform_(low, high)
+            if torch.finfo(param.data.dtype).bits < 16:
```

Comment:
Can you add a comment saying we do this because uniform doesn't support <8-bit data type?

Reply:
Sure, Added a comment :) Thanks